controllers/azurejson_machine_controller.go

```diff
@@ -126,6 +150,9 @@ func (r *AzureJSONMachineReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		return reconcile.Result{}, err
 	}
 
+	log.Info("Reconciling AzureMachine", "name", azureMachine.Name, "namespace", azureMachine.Namespace)
```

Comment:
I'll remove the log statements when this is ready for review.

Reply:
I think we're ready!